backend/src/seeds/graphql/mutations.ts

```diff
@@ -105,12 +110,12 @@ export const unDeleteUser = gql`
   }
 `
 
-export const createPendingCreations = gql`
-  mutation ($pendingCreations: [CreatePendingCreationArgs!]!) {
-    createPendingCreations(pendingCreations: $pendingCreations) {
+export const adminCreateContributions = gql`
```

Comment:
please do not mix Rights/Rules with method names, especially when with replacement of the elopage process the user will capture his contributions himself and the admin will only confirm them